packages/commonwealth/client/scripts/views/components/KnockNotifications/CustomNotificationCell.tsx

```diff
@@ -0,0 +1,29 @@
+import { Avatar } from '@knocklabs/react';
+import '@knocklabs/react-notification-feed/dist/index.css';
+import moment from 'moment';
+import React from 'react';
+import { CWText } from '../component_kit/cw_text';
+
+// eslint-disable-next-line react/no-multi-comp
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
1. react/no-multi-comp => is it necessary?
2. add type to `item`, at least minimal what is used in the component